src/app/components/annotations/Annotation.js

```diff
@@ -77,7 +76,7 @@ class Annotation extends Component {
       ? <span className={cx('test-annotation__timestamp', styles['annotation-timestamp'])}><TimeBefore date={updatedAt} /></span>
       : null;
     const authorName = activity.user
-      ? <ProfileLink className="annotation__author-name" user={activity.user} /> : null;
+      ? <span className="annotation__author-name" user={activity.user}> {activity.user.name} </span> : null;
```

Comment:
The `user` prop only makes sense as passed to `ProfileLink` which expects a `user` prop. For the  `span` element you should pass the  `className` only.

Reply:
Makes sense! 😅